DeviceSDK/src/main/java/org/astarteplatform/devicesdk/AstarteDeviceIdHelper.java

```diff
@@ -0,0 +1,25 @@
+package org.astarteplatform.devicesdk;
+
+import java.nio.ByteBuffer;
+import java.util.Base64;
+import java.util.UUID;
+import org.astarteplatform.devicesdk.util.UUID5;
+
+public class AstarteDeviceIdHelper {
+
+  public static String generateId() {
+    UUID randomUUID = UUID.randomUUID();
+    Base64.Encoder encoder = Base64.getUrlEncoder().withoutPadding();
+    return encoder.encodeToString(randomUUID.toString().getBytes());
+  }
+
+  public static String generateId(UUID namespace, String deviceId) {
```

Comment:
Change `deviceId` to `uniqueData` (the `deviceId` it's what it's returned from the function)